pkg/controller/instanceset/pod_role_event_handler.go

```diff
@@ -74,7 +74,7 @@ func (h *PodRoleEventHandler) Handle(cli client.Client, reqCtx intctrlutil.Reque
 	// HACK: to support kb-agent probe event
 	event = h.transformKBAgentProbeEvent(reqCtx.Log, event)
 
-	filePaths := []string{readinessProbeEventFieldPath, legacyEventFieldPath, lorryEventFieldPath}
```

Comment:
Do not remove it, the PodRoleEventHandler for version 1.0 is still needed to support older instances.

Reply:
Can old instances automatically upgrade to kb-agent?

Reply:
It depends on the user upgrading their cluster to v1 first.